src/main/resources/reference.conf

```diff
@@ -6,20 +6,29 @@ keen {
     master-key = ${?KEEN_MASTER_KEY}
     read-key = ${?KEEN_READ_KEY}
     write-key = ${?KEEN_WRITE_KEY}
+  }
+
+  # Queue and flushing behavior for writer clients when batching
+  queue {
+    batch {
+      size = 500
+      size = ${?KEEN_EVENT_QUEUE_BATCH_SIZE}
 
-    # configuration for local event store queue
-    queue {
-      batch {
-        size = ${?KEEN_EVENT_QUEUE_BATCH_SIZE}
-        timeout = ${?KEEN_EVENT_QUEUE_BATCH_TIMEOUT}
-      }
-      max-events-per-collection = ${?KEEN_EVENT_QUEUE_MAX_EVENTS_PER_COLLECTION}
-      send-interval {
-        events = ${?KEEN_EVENT_QUEUE_SEND_INTERVAL_EVENTS}
-        seconds = ${?KEEN_EVENT_QUEUE_SEND_INTERVAL_SECONDS}
-      }
-      shutdown-delay = ${?KEEN_EVENT_QUEUE_SHUTDOWN_DELAY}
+      timeout = 5 seconds
```

Comment:
Might be better to reference like so:

```scala
timeout = 5   # duration in seconds
```
or something to that effect so as to eliminate type confusion.